src/index.d.ts

```diff
@@ -1,7 +1,8 @@
+type Function = (...args: any[]) => any;
```

Comment:
I would rather not call that type ``Function`` as it is a Javascript constructor